### PR TITLE
chore: Annotate the main author(s) of each Java file

### PR DESCRIPTION
**Pull request by jens1o**

### CODE REVIEW

**Comment by jens1o** [comment]
Please note that I checked who has mainly contributed on a source-code level basis.

Hence, refactoring without any new developed feature or enhancement does not mean that someone will automatically get author of one's class.

**Review by Jonas-Karl — COMMENTED** [review]
Fine for me

File: src/test/java/de/dhbw/karlsruhe/students/mailflow/unit/JSONParserTest.java [hunk 1]
```diff
@@ -26,108 +29,102 @@ void successfullyParseJsonToMailbox() {
...
+                  "date": "2024-03-21T00:30:38.809547400"
                 }
-              ],
-              "type": "READ",
+              },
+              "content": "someContent",
+              "attachments": [],
               "id": {
-                "id": "2d1e184a-6120-4d15-918b-05c38382a4f9"
+                "id": "81ef6faa-86bd-441e-b76a-3d37c7227d31"
```
[the remark this answers is not in the document]
Comment (jens1o) [reply src/test/java/de/dhbw/karlsruhe/students/mailflow/unit/JSONParserTest.java:91]: it's still there: https://github.com/DHBW-Mailflow/mailserver/pull/96/files/302226b6a3de92eb6b82015d9647da16a4db57d3#diff-ca7865eff9d06b012fbe6ac99fb888af1145579471e952b112b897458cccdd4cR97

You're just looking at a different part of the diff when comparing side-by-side due to the formatting-changes.

**Review by Jonas-Karl — APPROVED** [review]

**Review by seiferla — APPROVED** [review]
in germany we say "isch approved"

**Pull request merged**